packages/api/hardware-checkout/src/services/hardware.mjs

```diff
@@ -0,0 +1,96 @@
+import database from '../../../core/src/utils/database.js';
+
+const hardware = (app) => {
+    
+    //GET operatiion to return a piece of hardware by unique ID 
+    //This function executes a stored procedure the returns a JSON object containing details about the hardware item 
+    //const scheme is the JSON object that is returned containing the details of the hardware item
+    app.get('/hardware/:id', async (req, res) => {
+        const hardware = await database.executeStoredProcedure('Select_Single_Hardware_Items', req.params.id);
+        const scheme = {
```

Comment:
After the above change, you should be able to change you code below to:
```
id: hardware[0],
tag: hardware[1],
. . . rest of the attributes
```

Also, just for consistency, please remove the quotes from the object keys.